src/only-export-components.test.ts

```diff
@@ -189,6 +189,21 @@ const valid = [
     name: "Only React context",
     code: "export const MyContext = createContext('test');",
   },
+  {
+    name: "Component and local React Context from other module",
+    code: "export const MyComponent = () => {}; const MyContext = createMyContext('test');",
+    errorId: "reactContext",
+    options: [
+      {
+        createContextMethods: ["createMyContext"],
+      },
+    ],
+  },
```

Comment:
We can drop this case, this case of Component + local is already covers multiple times